packages/components/src/organisms/Navbar/Navbar.tsx

```diff
@@ -17,15 +17,15 @@ const Navbar = forwardRef<HTMLDivElement, NavbarProps>(function Navbar(
   ref
 ) {
   return (
-    <section
+    <header
```

Comment:
hmm, I think we could keep section and make the `section/navbar` wrapper as header

Reply:
I changed that directly on `@faststore/components` to be a header instead a section ✅